src/posts/xarray-biology/index.md

```diff
@@ -0,0 +1,148 @@
+---
+title: 'Accelerating Biological Analysis with Xarray: Why You Should Try Using It Today'
+date: '2025-05-04'
+authors:
+  - name: Ian Hunt-Isaak
+    github: ianhi
+summary: 'A discussion of how Xarray fits into Biological analysis workflows'
+---
+
+I'm Ian, a multimodal microscopist, and the new "Xarray Community Developer." In this role, funded by the Chan Zuckerberg Institute, I work full-time to support the usage of Xarray in biological and biomedical applications.
+
+I ended up here because, five years in graduate school, I was sitting at my computer frustratedly trying to keep track of microscopy image metadata as I passed images through complicated, larger-than-memory jobs on the cluster. Which axis was what? Where did the exposure times go? I know I changed the media at 32 minutes, but what time index was that? Desperate for a solution, I found `Xarray`, seemingly a solution to all my woes. But I left thinking it was not for me as the examples were geoscience-centric. Fortunately, I eventually rediscovered `Xarray`, mentally translated the examples to biology, and found it incredibly useful. It is so useful that it since served as my default tool for all analyses in both structural and microbiology.
```

Comment:
Let's just use Xarray, NumPy etc. and not stick the project names in `code` formatting